.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Issues covered
+Add reference(s) to a related issue in your repository.
+
+## Description
+A description of the changes proposed in the pull request. Explain what it does to give the reviewer some context for what they're reviewing.
+
+## How to test
+1. Navigate to...
+2. Tap...
+Any testing instruction
```

Comment:
```suggestion
```

Reply:
If we are going to write down test steps maybe it makes sense to copy them to the ticket somehow so @setch-l can also use them for UAT?

Reply:
idk I don't necessarily want to create a heavier process or have to manually copy text and then have it duplicated in two places 🤔 maybe I regret making this comment.